core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5035,7 +5034,7 @@ components:
       type: object
       properties:
         value:
-          type: string
+          type: object
```

Comment:
@jgainerdewar you may be the best person to answer this even though this was in here [3 name changes + 9 years ago](https://github.com/broadinstitute/rawls/commit/cff3543cc983e89136126aa90a270f9cd6f7160c) 😂.  Maybe at the time the only expression was in fact a string and at some point arrays were added and this wasnt updated to match?

Reply:
LGTM assuming you can see in API responses that we aren't string-ifying more complex types.